data-loader/core/src/main/resources/config.properties

```diff
@@ -0,0 +1,2 @@
+transaction.batch.thread.pool.size=16
+import.data.chunk.queue.size=256
```

Comment:
[minor] A file should end with a newline https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
I have added a new line.
Thank you.